src/pages/macros.tsx

```diff
@@ -0,0 +1,283 @@
+import { useDocumentTitle } from 'usehooks-ts';
+import { Table, TableColumn, TableRow } from 'components/Table';
+import { faPlayCircle, faSkull } from '@fortawesome/free-solid-svg-icons';
+import { ButtonMenuConfig } from 'components/ButtonMenu';
+import {
+  getMacros,
+  getTasks,
+  getInstanceHistory,
+  createTask,
+  killTask,
+} from 'utils/apis';
+import { InstanceContext } from 'data/InstanceContext';
+import { useContext, useEffect, useState, useMemo } from 'react';
+import { MacroEntry } from 'bindings/MacroEntry';
+import clsx from 'clsx';
+import { useQueryClient } from '@tanstack/react-query';
+import { toast } from 'react-toastify';
+
+export type MacrosPage = 'All Macros' | 'Running Tasks' | 'History';
+const Macros = () => {
+  useDocumentTitle('Macros');
```

Comment:
That's good but we don't have `useDocumentTitle` anywhere else, so after the user goes to Macro page once, their document title is set to "Macros" forever.
Let's remove this for now and fix it in https://github.com/Lodestone-Team/lodestone/issues/241